source/assets/sass/vendor/_syntax-accessible.scss

```diff
@@ -1,3 +1,5 @@
+// @@@ These definitions no longer match the added classes...
```

Comment:
These are the existing styles for the old site, but the tokens no longer match because we're using Prism instead of [middleman-syntax](https://github.com/middleman/middleman-syntax) (with a modified [Rouge github theme](https://spsarolkar.github.io/rouge-theme-preview/)).

Reply:
I wonder if we can take an existing prism scheme, and swap in some colors to get close

Reply:
I think that's roughly what we did on OddLeventy: https://github.com/oddbird/oddleventy/blob/main/src/scss/patterns/_code.scss

Reply:
I ended up taking the the OddLeventy approach here, swapping out colors to match an existing prism theme.